tests/test_MIDIMessage_unittests.py

```diff
@@ -29,7 +31,9 @@
 # pylint: enable=wrong-import-position
 
 
-class Test_MIDIMessage_from_message_byte_tests(unittest.TestCase):
+class Test_MIDIMessage_from_message_byte_tests(
+    unittest.TestCase
+):  # pylint: disable=invalid-name
```

Comment:
@kattni In cases where the invalid-name warnings are for classes and methods, is this the correct way to do it? I didn't think it'd be a good idea to change the names since that would break existing code, but I'd be fine doing that if it's the correct way to do it.